### PR TITLE
Remove offensive from the rule, too subjective

### PR DESCRIPTION
**Pull request by campbecf**
As worded you could be reported to a moderator for offending someone and the language in here describes anything as being offensive as being unacceptable. Being offended is so subjective in a diverse community that this could be hard to correctly interpret. The other clauses in this statement are much more specific.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## Unacceptable Behavior
 
-- Conduct or speech which might be considered sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory or offensive in nature.
```
Comment (zarahzachz) [inline README.md:17]: I'd remove "might be" as well

**Comment by geofflane** [comment]
Subjectivity is not a flaw. Yes, different people will be offended by different things. The goal is to create an inclusive community and this give people a mechanism to raise those concerns to the moderators. The moderators will talk to the parties involved. It opens dialogue and maybe we'll learn something new.

**Comment by campbecf** [comment]
Being subjective without an established precedent or context can be a flaw when it comes to guidelines or rules we expect people to abide by. It can be impossible to know what would offend someone. Being offended is not always a bad thing. 

If you tell me it was a horrible idea to write my own encryption, I might be offended but I should be.

By leaving the interpretation of offense up to the individual moderators we will have uneven enforcement of what is and is not offensive. 

If we are going to keep this part I would advocate we create both positive and negative examples of statements that are/are not offensive to provide clarity and context.

**Comment by campbecf** [comment]
I would prefer to use #9 instead of making these changes.

**Comment by campbecf** [comment]
Closing this in favor of using the Geek Feminism Code of Conduct based on our discussions on Slack.

**Pull request closed without merging**